cassandra nodetool_test.py::TestNodetool::test_nodetool_timeout_commands true true
the_test is nodetool_test.py::TestNodetool::test_nodetool_timeout_commands
============================= test session starts ==============================
platform linux -- Python 3.6.9, pytest-3.6.4, py-1.11.0, pluggy-0.7.1 -- /users/masix/dtest/bin/python
cachedir: .pytest_cache
rootdir: /users/masix/cassandra-dtest, inifile: pytest.ini
plugins: timeout-1.4.2, repeat-0.9.1, flaky-3.7.0
timeout: 900.0s
timeout method: signal
timeout func_only: False
collecting ... collected 1 item

nodetool_test.py::TestNodetool::test_nodetool_timeout_commands 
-------------------------------- live log call ---------------------------------
05:34:50,922 ccmlib.node WARNING [msx] ccmlib:copy_config_files.py, conf_dir = /users/masix/cassandra/conf
05:34:51,7 ccmlib.node WARNING [msx] ccmlib:node.py, self.name = node1
05:34:51,7 ccmlib.node WARNING [msx] ccmlib:node.py, sstable_preemptive_open_interval_in_mb = 50
05:34:51,76 ccmlib.node WARNING [msx] ccmlib:node.py, self.name = node1
05:34:51,76 ccmlib.node WARNING [msx] ccmlib:node.py, sstable_preemptive_open_interval_in_mb = 50
PASSED                                                                   [100%]
===Flaky Test Report===

test_nodetool_timeout_commands passed 1 out of the required 1 times. Success!

===End Flaky Test Report===

========================== 1 passed in 35.38 seconds ===========================
[msx] rc = 0
